app/src/main/java/com/example/sharingang/utils/DateHelper.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.sharingang.utils
+
+import java.util.*
+
+object DateHelper {
+
+    private val monthsMap = hashMapOf(
+        0 to "Jan",
+        1 to "Feb",
+        2 to "Mar",
+        3 to "Apr",
+        4 to "May",
+        5 to "Jun",
+        6 to "Jul",
+        7 to "Aug",
+        8 to "Sep",
+        9 to "Oct",
+        10 to "Nov",
+        11 to "Dec"
```

Comment:
Could you use a string resource instead?

Reply:
I don't know how to do that, as string resources can only be accessed only from a Context.

Reply:
[https://stackoverflow.com/questions/4253328/getstring-outside-of-a-context-or-activity](url)

Reply:
I have just looked up the solutions, however, none of them could solve the problem. Apparently, only the system resources are accessible with the suggested method, not the ones bound to the application.

Reply:
Could we use system resources then?

Reply:
The system resources are not modifiable because they are provided by the SDK, not the app. I don't think there was any other solution than using `DateHelper` as a Class, not an Object, and providing it with the context. So that's what I did